scripts/build-vpp.sh

```diff
@@ -2,11 +2,18 @@
 set -e
 
 VPP_CACHE_DIR=$HOME/build-cache/vpp
-VPP_COMMIT="92b0275a364022af6ab828dfac83e38c0117cfe6"
-VPP_IMG_TAG="92b0275"
+
+VPP_COMMIT=`git submodule status vpp | awk '{print $1}'`
```

Comment:
On the one hand this is definitely better solution than hardcoding the commit id, on the other we reuse this script in `contiv/vpp` project where vpp-agent is in `vendor` dir, thus there is no submodule. 

What about supporting "fallback" variable (i.e. allow to define commit manualy) if there is no submodule?

Reply:
Using fallback variable only if there is no submodule would defeat the intended purpose of preventing commits where variable and submodule don't match. Then it would be better to use the hard coded variable as before to not confuse anyone.

As I was looking for the most suitable solution I could not find any real usage or dependents for the vpp submodule. Is there even any?